packages/css-library/src/stylesheets/core.scss

```diff
@@ -15,6 +15,11 @@
 //============ PORTED VARIABLES ==============//
 @import './formation-overrides/variables';
 
+// ----- VA GENERAL (MAIN FILE/OTHER) ---- //
+@import 'base/va';
+@import 'base/fonts';
+@import 'base/utils';
```

Comment:
Moved these higher because we likely want any formation override stylesheets to take precedent